precompiles/gov/IGov.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.17;
+
+/// @dev The IGov contract's address.
+address constant GOV_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000805;
+
+/// @dev Define all the available gov methods.
+string constant MSG_VOTE = "/cosmos.gov.v1.MsgVote";
```

Comment:
what's this for?

Reply:
in other precompile these are included to make it easier for devs to get the msgUrl for approvals.
In this case, we're not including any auth logic, so we can remove it for now

Reply:
yes, please remove it